project/OSGi.scala

```diff
@@ -75,13 +75,14 @@ object OSGi {
   private lazy val akkaParboiledImport = projectVersioned("akka.parboiled2.*")
   private lazy val akkaShapelessImport = projectVersioned("akka.shapeless.*")
   private lazy val akkaMacrosImport = projectVersioned("akka.macros.*")
-  private lazy val akkaImport = "akka," + patchVersioned("akka.*")
-  private lazy val configImport = patchVersioned("com.typesafe.config.*")
-  private lazy val scalaImport = minorVersioned("scala.*")
+  private lazy val akkaImport = "akka," + patchVersionToNextMajorVersion("akka.*")
+  private lazy val configImport = patchVersionToNextMinorVersion("com.typesafe.config.*")
+  private lazy val scalaImport = minorVersionToNextMinorVersion("scala.*")
```

Comment:
I think this means "until 2.13 but excluding that one" which means all patches right @michael-smith ?